test-case/check-alsabat.sh

```diff
@@ -25,22 +25,25 @@ OPT_NAME['p']='pcm_p'     	OPT_DESC['p']='pcm for playback. Example: hw:0,0'
 OPT_HAS_ARG['p']=1          	OPT_VAL['p']=''
 
 OPT_NAME['C']='channel_c'       OPT_DESC['C']='channel number for capture.'
-OPT_HAS_ARG['C']=1             OPT_VAL['C']='1'
+OPT_HAS_ARG['C']=1              OPT_VAL['C']='1'
 
 OPT_NAME['r']='rate'            OPT_DESC['r']='sample rate'
-OPT_HAS_ARG['r']=1             OPT_VAL['r']=48000
+OPT_HAS_ARG['r']=1              OPT_VAL['r']=48000
```

Comment:
This is pure git noise, don't.
https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes

Reply:
Updated.